azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -492,6 +512,19 @@ def az_cli(self):
     def token_provider(self):
         return self._token_provider
 
+    @property

```

Comment:
consider annotating return types

Reply:
out of the scope of this PR.
Plus it will only conflict with your changes more if I annotate APIs which you've annotated in your PR